cloud/passportauth/example/public/login.html

```diff
@@ -0,0 +1,73 @@
+<html xmlns="http://www.w3.org/1999/xhtml" class="">
```

Comment:
As agreed let's use simple login in example and have proper shinny one in demo app.

ping @austincunningham